### PR TITLE
Add n8n icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/7993

**Popularity metric:**

The Similarwebrank is 60,987. See https://www.similarweb.com/fr/website/n8n.io

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I forgot where I downloaded the original svg icon (had it on my computer for quite a while). I just combined the paths and applied the transformations to them. Then optimized it.

### CODE REVIEW

File: icons/n8n.svg
Comment (uncenter) [inline icons/n8n.svg:1]: image

I'm seeing a bunch of differences as illustrated by the screenshot above. Are you using the logo from their website's header? Also I'd like to note that you said " I just combined the paths and applied the transformations to them." - there are no paths to combine for this logo!

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/n8n.svg
Comment (uncenter) [reply icons/n8n.svg:1]: They also have a press kit from which you can source the icon: https://n8n.io/press/. `n8n-symbol-only.svg` would be the file to grab.

Comment (CodeShakingSheep) [reply icons/n8n.svg:1]: Hm, right. As I wrote I had this other n8n icon svg on my computer for a while (and don't know anymore from where I downloaded it). With the official svg icon you linked, I have a problem with removing `fill-rule="evenodd" clip-rule="evenodd"`, see screenshot. The problem is that 2 of the small circles get filled. I don't know how to solve that. Do you have any idea?

image

Comment (uncenter) [reply icons/n8n.svg:1]: Hmm I'm not sure. @PeterShaggyNoble I recall you helped me before with an icon that had `fill-rule="evenodd"`, how can we get rid of it here?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10074,6 +10074,11 @@
             "hex": "48AC98",
             "source": "https://n26.com"
         },
+        {
+            "title": "n8n",
+            "hex": "EA4B71",
+            "source": "https://n8n.io"
```
Comment (adamrusted) [inline _data/simple-icons.json:10080]:
```suggestion
            "source": "https://n8n.io/press"
```

File: icons/n8n.svg
Comment (adamrusted) [reply icons/n8n.svg:1]: I don't know the exact science with this - but I've always been able to 'split' the path in my editor, move the circle shapes up/down (changes per program) and then re-do the merging.

Comment (CodeShakingSheep) [reply icons/n8n.svg:1]: Ok, I managed to solve this by breaking apart the path (in Inkscape), then creating 4 little circles in the size of the 4 small paths and then took the difference between the big path and the 4 little circles respectively. Is it good now?

**Review by uncenter — APPROVED** [review]

**Review by adamrusted — APPROVED** [review]

**Pull request merged**